CHANGELOG.md

```diff
@@ -1,2 +1,4 @@
+## **0.2.1**
```

Comment:
Do we still need this file? We stripped them out of the other repos, or are we keeping them as this is OSS? @kevinhodges